sample.yml

```diff
@@ -4,6 +4,19 @@
 # on components in your cluster!
 - include: "{{ playbook_dir }}/playbooks/check-requirements.yml"
 
+# ENABLE REPOS - all of our packages are installed from these repos, we need to
+# enable them before any roles can be used.
+- hosts: all
+  tasks:
+    - name: enable yum repos
+      sudo: yes
+      copy:
+        src: "roles/common/files/{{ item }}"
+        dest: "/etc/yum.repos.d/{{ item }}"
+      with_items:
+        - asteris-mantl-rpm.repo
+        - ciscocloud-rpm.repo
+
 # BASICS - we need every node in the cluster to have common software running to
```

Comment:
I don't think this belongs in the playbook. We should include roles and depend on them, sure, but this pins us to supporting this for longer than in a role. (Because it's encouraged to update your `sample.yml` with customizations.)

Maybe this should be a "repos" role that common et al. depend on.